src/plus/integrations/integrationService.ts

```diff
@@ -44,21 +50,71 @@ export class IntegrationService implements Disposable {
 	private readonly _disposable: Disposable;
 	private _integrations = new Map<IntegrationKey, Integration>();
 
-	constructor(private readonly container: Container) {
+	constructor(
+		private readonly container: Container,
+		private readonly connection: ServerConnection,
+	) {
 		this._disposable = Disposable.from(
 			configuration.onDidChange(e => {
 				if (configuration.changed(e, 'remotes')) {
 					this._ignoreSSLErrors.clear();
 				}
 			}),
 			authentication.onDidChangeSessions(this.onAuthenticationSessionsChanged, this),
+			container.subscription.onDidCheckIn(this.onUserCheckedIn, this),
+			...this.registerCommands(),
 		);
 	}
 
 	dispose() {
 		this._disposable?.dispose();
 	}
 
+	private registerCommands(): Disposable[] {
+		return [registerCommand(Commands.PlusManageCloudIntegrations, () => this.manageCloudIntegrations())];
+	}
+
```

Comment:
Since this service is delay loaded (not until access), we'll need to move the command elsewhere -- probably just create a new command similar to https://github.com/gitkraken/vscode-gitlens/blob/4fa1227c8c99039de09d4b911bd3f84beae0ddeb/src/commands/resets.ts#L9-L18